fastlane/testflight.md

```diff
@@ -151,10 +153,11 @@ _Referring to the table below, tap on each **IDENTIFIER** that has a different *
 
 #### Table of Identifiers
```

Comment:
Please modify this heading to 3 `#` instead of 4 `#`.

(We are planning that change in trio-docs).

Reply:
`#` removed.